src/schema/v2/artist/meta.ts

```diff
@@ -93,505 +89,3 @@ const Meta: GraphQLFieldConfig<void, ResolverContext> = {
 }
 
 export default Meta
-
-// https://artsyproduct.atlassian.net/browse/DIA-1131
-const EXPERIMENT_GROUP = [
```

Comment:
oh interesting. I wonder how we determine which artists go into this control group?